TrackMyStuffApp/Program.cs

```diff
@@ -11,10 +11,14 @@ public class Program
     {
         public static void Main(string[] args)
         {
-            // Create configuration object from appsettings.json file.
+            // Create configuration object from appsettings.json file
+            // and it will be overridden by appsetting.{ENVIRONMENT}.json file.
+            var enviroment = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
```

Comment:
This get the current Environment Variables for this app. Local development already set by `launchSettings.json` file.

For  deployment to AWS, we need set the Environment Property as following:

- Click on the _TrackMyStuff-dev_ environment.
- On the left-hand side, click on _Conguration_.
- Under _Category_ => _Software_ . Then click _Edit_.
- Under _Environment properties_, add the following:
   - `ASPNETCORE_ENVIRONMENT` for name
   - `Production` for Value.
- Apply.